### PR TITLE
feat(user): add user name

### PR DESCRIPTION
**Pull request by andreybakanovsky**
and update the app in necessary places

### CODE REVIEW

File: app/mailers/invitation_mailer.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 class InvitationMailer < ApplicationMailer
-  def account_invitation(account_invitation)
+  def account_invitation(account_invitation, current_user)
```
Comment (ka8725) [inline app/mailers/invitation_mailer.rb:4]:
```suggestion
  def account_invitation(account_invitation)
```

the user can be taken from `account_invitation`

Comment (ka8725) [reply app/mailers/invitation_mailer.rb:4]: @andreybakanovsky what about the user_id column inside account_invitation model? Isn't it the user who sent the invite?

Comment (andreybakanovsky) [reply app/mailers/invitation_mailer.rb:4]: Got it.

**Review by VladislavSokov — APPROVED** [review]

**Review by ka8725 — APPROVED** [review]

**Comment by ka8725** [comment]
@andreybakanovsky please fix the failed test

**Pull request merged**